frontend/src/v5/ui/routes/dashboard/projects/projectsList/createProjectModal/createProjectModal.component.tsx

```diff
@@ -54,15 +54,14 @@ export const CreateProjectModal = ({ open, onClickClose }: CreateProjectModalPro
 		trigger,
 	} = useForm<IFormInput>({
 		mode: 'onChange',
-		resolver: yupResolver(CreateProjectSchema),
-		context: { existingProjects },
+		resolver: yupResolver(ProjectSchema),
+		context: { existingNames },
 		defaultValues: DEFAULT_VALUES,
 	});
 
 	const onSubmissionError = (error) => {
 		if (projectAlreadyExists(error)) {
-			const { projectName } = getValues();
-			setExistingProjects((currentValue) => [...currentValue, projectName]);
+			setExistingNames((currentValue) => [...currentValue, getValues('projectName').toLocaleLowerCase()]);
```

Comment:
This is not necessary, the comparison is already changing the case projectSchemes